include/mitsuba/render/film.h

```diff
@@ -31,6 +31,9 @@ class MTS_EXPORT_RENDER Film : public Object {
     /// Develop the film and write the result to the previously specified filename
     virtual void develop() = 0;
 
+    /// Overwrite the weight channel to the given value
```

Comment:
Can we remove this interface? Mitsuba 1 did not need this to implement the particle tracer.

Reply:
Mitsuba 1 used that API instead (pretty similar). Do you prefer the old one?

https://github.com/mitsuba-renderer/mitsuba/blob/cfeb7766e7a1513492451f35dc65b86409655a7b/include/mitsuba/render/film.h#L54-L55